lua/pac3/core/client/parts/event.lua

```diff
@@ -3382,10 +3382,6 @@ pac.AddHook("EntityFireBullets", "firebullets", function(ent, data)
 	ent.pac_fire_bullets = {name = data.AmmoType, time = pac.RealTime, reset = true}
 
 	pac.CallRecursiveOnAllParts("OnFireBullets")
-
-	if ent.pac_hide_bullets then
-		return false
-	end
```

Comment:
Idk if removing this is right. Someone else verify

Reply:
At first glance, if we search "pac_hide_bullets", it seemed as though the field isn't created anywhere (so, accessing it would always be nil if that was the case)
But, if we run the search without the pac_ prefix, it leads to EntityField functions, the builder thing that set an entity field with that prefix.
(for OP's information) This one wasn't obvious, that prefix business is just a thing that we do a few times throughout the code.

We still want that return because we still want the option to hide the bullets.

But there's some side issues in singleplayer. I tried to test, it appears the EntityFireBullets hook is ineffective. But it's fine with local server and p2p setups for some reason... oh well!

Reply:
@unknao can you revert this change?